onnxruntime/contrib_ops/cuda/fused_conv.cc

```diff
@@ -120,6 +124,15 @@ ONNX_OPERATOR_TYPED_KERNEL_EX(
     (*KernelDefBuilder::Create()).TypeConstraint("T", DataTypeImpl::GetTensorType<float>()),
     FusedConv<float>);
 
+ONNX_OPERATOR_TYPED_KERNEL_EX(
+    FusedConv,
```

Comment:
I am guessing this is an enhancement to the existing `FusedConv` kernel and not an NHWC change per se ?

Can you please add tests for the fp16 support and new activations support here - https://github.com/microsoft/onnxruntime/blob/dd3b2cefd6abdd6a0fe6f6701d26d1299f87f1fa/onnxruntime/test/contrib_ops/fused_conv_test.cc#L38 ?

Reply:
Also, needs updates here - https://github.com/microsoft/onnxruntime/blob/dd3b2cefd6abdd6a0fe6f6701d26d1299f87f1fa/onnxruntime/core/optimizer/conv_activation_fusion.cc#L54

Reply:
done.